src/styles/watch.scss

```diff
@@ -0,0 +1,51 @@
+@keyframes rotate {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.watch {
+  @include circle($watch-size);
+  display: flex;
+  box-sizing: border-box;
+  justify-content: center;
+  align-items: center;
+  background-image: url(/images/cp.jpg);
+
+  &__minute-hand {
+    position: absolute;
+    box-sizing: border-box;
+    bottom: 50%;
+    width: 2vmin;
+    height: 20vmin;
+    background: #e66465;
+    border-radius: 50%;
+    border: 0.5vmin solid black;
+    transform-origin: center bottom;
+    animation: rotate 60s linear infinite;
+  }
+
+  &__hour-hand {
+    position: absolute;
+    box-sizing: border-box;
+    bottom: 50%;
+    width: 2vmin;
+    height: 40vmin;
+    background: #9198e5;
+    border-radius: 50%;
+    border: 0.5vmin solid black;
+    transform-origin: center bottom;
+    animation: rotate 5s linear infinite;
+  }
```

Comment:
I see some common styles heare for a **&__hand**. You could create additional class for styling it like:
```css
&__hand {
    position: absolute;
    box-sizing: border-box;
    bottom: 50%;
    width: 2vmin;
    border-radius: 50%;
    border: 0.5vmin solid black;
    transform-origin: center bottom;

   &-hour {
      height: 20vmin;
      background: #9198e5;
      animation: rotate 60s linear infinite;
   }

   &-minute{
      height: 40vmin;
      background: #e66465;
      animation: rotate 5s linear infinite;
   }
}
```

I would also change the classes for **&__hand-hour** ,**&__hand-minute**.
And I think minute hand is usually longer than hour hand so I have  swap heights and animation time:)